context.go

```diff
@@ -55,9 +55,15 @@ type MessageEvent struct {
 	// Channel ID where the message was sent
 	Channel string
 
+	// ChannelName where the message was sent
+	ChannelName func() string
```

Comment:
All the other struct fields are strings, for the sake of consistency ChannelName and UserName should be as well.

Reply:
https://github.com/shomali11/slacker/pull/100#pullrequestreview-1019743388